be/src/runtime/cluster_id_mgr.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
header & license? 

Plus: what is `Clustder Id Manager`?  Why we need this class?

Reply:
Or is `ClusterIdManager` just for refactor?

Reply:
> Or is `ClusterIdManager` just for refactor?

It is created for EmptyStorageEngine, which has no DataDir, so it employ ClusterIdManager directly.